### PR TITLE
Added macros to Main.cc to make adding a Python module now a one line…

### PR DESCRIPTION
**Pull request by macd**
… change. Also changed the name odbpy to odb_py to matching the naming convention of the other Python modules.

Signed-off-by: Don MacMillen <don@macmillen.net>

### CODE REVIEW

File: src/Main.cc [hunk 1]
```diff
@@ -79,13 +79,21 @@ using sta::stringEq;
 using std::string;
 
 #ifdef ENABLE_PYTHON3
+#define PY_MODULES_WITHOUT_OPENROAD \
```
Comment (maliberty) [inline src/Main.cc:82]: This should take X as an arg

File: src/Main.cc [hunk 2]
```diff
@@ -79,13 +79,21 @@ using sta::stringEq;
...
+  X(ifp)                            \
+  X(utl)                            \
+  X(ant)                            \
+  X(grt)                            \
+  X(odb)
+
+#define PY_MODULES            \
```
Comment (maliberty) [inline src/Main.cc:89]: This too

File: src/Main.cc (hunk 1 above)
Comment (maliberty) [reply src/Main.cc:82]: eg https://github.com/The-OpenROAD-Project/OpenROAD/blob/846c27706460b3bbce5d584eaab8c2bec8a3d49b/src/utl/include/utl/Logger.h#L58

Comment (macd) [reply src/Main.cc:82]: Yes, that is less obscure. Will do.

**Review by maliberty — APPROVED** [review]

**Pull request merged**